agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java

```diff
@@ -355,7 +355,7 @@ private DiseaseDownloadRow getBaseDiseaseDownloadRow(DiseaseAnnotationDocument a
 			for (org.alliancegenome.curation_api.model.entities.DiseaseAnnotation da : annotation.getPrimaryAnnotations()) {
 				if (CollectionUtils.isNotEmpty(da.getConditionRelations())) {
 					List<ConditionRelation> conditionModifiers = da.getConditionRelations().stream().filter(conditionRelation -> conditionRelation.getConditionRelationType() != null)
-						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced")).toList();
+						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced") || conditionRelation.getConditionRelationType().getName().contains("ameliorated") || conditionRelation.getConditionRelationType().getName().contains("exacerbated")).toList();
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 400 characters (found 410).
<!-- __reviewdog__:ChBkNDNmNmFiOTc1Nzk3Y2NiEgpjaGVja3N0eWxl -->